src/flavors/react/components/Link/index.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+interface LinkProps {
+  /** Define the link text */
+  text: string;
+  /** Define where the link will redirect */
+  url: string;
+}
+
+const Link: React.FC<LinkProps> = ({ text, url }) => {
+  return (
+    <a href={url} className='iq-link' target="_blank" rel="noopener">
+      {text}
+    </a>
```

Comment:
boaa, só uns detalhezinhos que acho que pode ser uma boa darmos uma olhada aqui são:
* Repassar o `...props` para o `a`:  podemos passar um `id`, data attribute  ou qualquer outro atributo do html pro `a` se for necessário em algum projeto.
* acho que é uma boa não deixar o target fixo como `_blank` pq nem todo link necessariamente vai ser aberto em outra aba
* Talvez usar a prop `children` invés do `text` pode ser mais interessante na hora de usa-lo em algum texto pq ai a gente mantem o padrão de como a tag `a` é usada no html, exemplo: 
```jsx 
<p>clique nesse <Link url="/uma-pagina">link</Link> </p>
```
```suggestion
const Link: React.FC<LinkProps> = ({ url,  children, ...props }) => {
  return (
    <a href={url} className='iq-link' {...props}>
      {children}
    </a>
```